src/ar/mangalek/build.gradle

```
@@ -3,7 +3,7 @@ ext {
     extClass = '.Mangalek'
     themePkg = 'madara'
     baseUrl = 'https://lekmanga.net'
-    overrideVersionCode = 5
+    overrideVersionCode = 6
```

should set `isNsfw`. I didn't find any, if you can't find any as well use:
```suggestion
    overrideVersionCode = 6
    isNsfw = false
```